tests/lang/core.pk

```diff
@@ -30,3 +29,22 @@ def fn(p1, p2, p3) end
 assert(fn.name == "fn")
 assert(fn.arity == 3)
 
+## Math functions
+from math import PI, sin, cos, tan
+
+assert(sin(0) == 0)
+assert(sin(PI/2) == 1)
+
+# NOTE, probably a TODO
+# On my system, the correct value is that, maybe we truncate?
```

Comment:
the sine, cosine values are irrational for rational numbers, and our `PI = 3.14159265358979323846` which is approximated to 20 decimal places, So the results would differ by a tiny fraction.

The better way to have a threshold value for approximation.
```ruby
threshold = 0.0000000000001

assert(abs(cos(PI/3) - 0.5) < threshold ) # abs from math
assert(abs(tan(PI/4) - 1.0) < threshold )
...
  assert(abs(sin(i) / cos(i) - tan(i)) < threshold)
```